components/collections/codepointtrie_builder/src/wasm.rs

```diff
@@ -52,14 +52,14 @@ where
 
     let construct_ucptrie = instance
         .exports
-        .get_native_function::<(u32, u32, i32, i32, i32, i32), i32>("construct_ucptrie")
+        .get_native_function::<(u32, u32, u32, u32, i32, i32), i32>("construct_ucptrie")
```

Comment:
```suggestion
        .get_native_function::<(u32, u32, u32, u32, *const u32, u32), i32>("construct_ucptrie")
```

Reply:
Actually I guess `WasmPtr<u32>` instead of `*const u32`.

Reply:
Done.